api.yaml

```diff
@@ -138,8 +138,8 @@ paths:
           request: >
             {
               "time": "#!index .Request.Query.time 0!#",
-              "count": #!if .Request.Query.count !# #!index .Request.Query.count 0!# #! else !# 100 #! end !# #!if .Request.Query.type !# ,
-              "type": "#!index .Request.Query.type 0!#" #! end !#
+              "count": #!if .Request.Query.count !# #!index .Request.Query.count 0!# #! else !# 100 #! end !# ,
```

Comment:
this is not really a suggestion, mostly posing the question: would this default value be better if embedded as a value inside the app? If in the app, it can be used immediately by anyone who uses this service. If left here in the api descriptor, it must be duplicated by anyone who uses it.

Reply:
Do we have a good example? It looks like a good idea to me.

Reply:
Not that I know of. One approach would be to have an attribute inside your handler for a default page size which is populated from an environment variable.